accounts/templates/logout.html

```diff
@@ -0,0 +1,7 @@
+<a id="openstax-logout" 
+   name="openstax" 
+   href="{% url 'logout' %}" 
+   id="openstax-logout">
+   <input type="button" value= "Logout you will still be logged into the Openstax OAuth2 Provider"/>
```

Comment:
My mistake.   If you loggout on the CMS site then you will be still logged into openstax-accounts server.